app/tests/test_account/test_services.py

```diff
@@ -0,0 +1,35 @@
+from account.services_views import create_tokens, generate_access_token_from_refresh
+
+
+class TestRefreshToken:
```

Comment:
you don't need the class, just use plain functions

Reply:
done